config/routes.rb

```diff
@@ -7,7 +7,9 @@
   resources :follow_requests
   resources :likes
   resources :photos
-  
+
```

Comment:
Don't forget to devise for users! I went and saw it was already in your code but we need to see that code in this PR